dash/App/Chains/Connection/index.js

```diff
@@ -6,6 +6,32 @@ import Dropdown from '../../../../resources/Components/Dropdown'
 import link from '../../../../resources/link'
 import svg from '../../../../resources/svg'
 
+const ConnectionIndicator = ({ connection }) => {
+  const isConnected = connection.status === 'connected'
+  const isLoading = connection.status === 'loading'
+  const isPending = connection.status === 'pending'
+  const isSyncing = connection.status === 'syncing'
+  let status = 'Bad'
+  if (isConnected) {
+    status = 'Good'
+  } else if (isLoading || isPending || isSyncing) {
+    status = 'Pending'
+  }
+
+  return <>
```

Comment:
do these brackets do anything if there's a top level div being returned?

Reply:
they do not :)